Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.13-alpine as builder
+FROM golang:alpine as builder
```

Comment:
shall we stick to a specific golang version when building image (like 1.18.3-alpine)? 

https://hub.docker.com/_/golang

Reply:
works for me :)